tools/ci_build/github/azure-pipelines/azure-pipelines-py-packaging.yml

```diff
@@ -343,3 +343,65 @@ jobs:
         ArtifactName: onnxruntime
 
     - template: templates/component-governance-component-detection-steps.yml
+
+- job: Linux_ARM_py_Wheels
+  timeoutInMinutes: 60
+  pool: 'Linux-CPU'
+  strategy:
+    matrix:
+      Py37:
+        python.include: '3.7m'
+        cp.tag: 'cp37-cp37m'
+      Py36:
+        python.include: '3.6m'
+        cp.tag: 'cp36-cp36m'
+      Py35:
+        python.include: '3.5m'
+        cp.tag: 'cp35-cp35m'
+  steps:
+    - task: CmdLine@2
+      inputs:
+        script: |
+          set -e -x
+          sudo rm -rf *
+          cd $(Build.SourcesDirectory)
+          git submodule update --init --recursive
```

Comment:
Is the line 368 needed?

Reply:
We can either do this or use `submodules: recursive`.